### ISSUE TITLE
Snap doesnt start on ubuntu 18.04 with wayland

### ISSUE THREAD

**Issue by Remarknl** [thomasnordquist/MQTT-Explorer#145]
Starting it from a terminal gives:
`(mqtt-explorer.bin:26930): Gtk-WARNING **: 17:02:23.483: cannot open display: :0`

**Comment by thomasnordquist** [comment]
Thanks for the report.
Do you know if the AppImage build works?

The common fix for this seems to be, to run the app with a `DISABLE_WAYLAND=1` environmental.

```
> export DISABLE_WAYLAND=1
> mqtt explorer

# Or
> DISABLE_WAYLAND=1 mqtt-explorer
```

**Comment by Remarknl** [comment]
Thanks for the quick reply!

The export solution works.
The appImage also works as expected.

**Comment by thomasnordquist** [comment]
Thanks for the feedback, I'll see what I can do to provide a permanent fix.

**Comment by thomasnordquist** [comment]
I released v0.3.1 in the snap store to fix the issue for all Ubuntu users.

**Comment by thomasnordquist** [comment]
Fix is: c543d0fb85a753eb4ee07a5eb6ba10a975f0059f
